app/assets/stylesheets/application.scss

```diff
@@ -1,16 +1,4 @@
-/*
- * This is a manifest file that'll be compiled into application.css, which will include all the files
- * listed below.
- *
- * Any CSS and SCSS file within this directory, lib/assets/stylesheets, or any plugin's
- * vendor/assets/stylesheets directory can be referenced here using a relative path.
- *
- * You're free to add application-wide styles to this file and they'll appear at the bottom of the
- * compiled file so the styles you add here take precedence over styles defined in any other CSS/SCSS
- * files in this directory. Styles in this file should be added after the last require_* statement.
- * It is generally better to create a new file per style scope.
- *
- *= require_tree .
- *= require_self
- */
+$govuk-image-url-function: 'image-url';
```

Comment:
Documentation for these: https://github.com/alphagov/govuk-frontend/blob/master/docs/installation/installing-with-npm.md#alternative-solution

Reply:
TODO: Does this make sense for production, do we want to use another method for assets?